### PR TITLE
Update remove functionality to work for clicking anywhere on poster

### PR DESCRIPTION
**Pull request by VictoriaFC**
### Changes
- update create poster function to add a single poster element to the html 
- before adding poster element add an event listener to it
- use deleteSavedPoster function as the event handler when a poster is double clicked 
- use currentTarget.id on the double click event object to determine which poster to remove from the html and the savedPosters array

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -158,22 +152,22 @@ function getRandomIndex(array) {
 function saveCurrentPoster() {
```
Comment (Rallen13) [inline src/main.js:152]: You can move the savedPostersGrid query selector variable inside the saveCurrentPoster function as it is a local variable.

File: src/main.js [hunk 2]
```diff
@@ -158,22 +152,22 @@ function getRandomIndex(array) {
...
   if (!posterSavedAlready()) {
     savedPosters.push(currentPoster)
+    savedPostersGrid.appendChild(generateMiniPosterHTML())
   }
-  displaySavedPosters()
 }
 
-function displaySavedPosters() {
```
Comment (Rallen13) [inline src/main.js:165]: Lets group the function by display and data for better readability.

**Review by Rallen13 — APPROVED** [review]
Made some re-factor comments to the code. Let me know what you think about these suggestions.

File: src/main.js (hunk 1 above)
Comment (VictoriaFC) [reply src/main.js:152]: I went ahead and made this update!

File: src/main.js (hunk 2 above)
Comment (VictoriaFC) [reply src/main.js:165]: Re-factored code to implement this!

**Comment by Rallen13** [comment]
Re-factor looks good. Functionality for deleting poster by clicking anywhere on poster also works well!

**Pull request merged**